modules/cluster/addons/kube-proxy.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       - command:
         - /bin/sh
         - -c
-        - kube-proxy --v=2 --config=/var/lib/kube-proxy-config/config
+        - kube-proxy --v=2 --config=/var/lib/kube-proxy-config/config --metrics-bind-address=0.0.0.0
```

Comment:
I think `--metrics-bind-address` needs the port too, so `--metrics-bind-address=0.0.0.0:10249`

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go#L117